recipe_scrapers/usapears.py

```diff
@@ -65,10 +77,19 @@ def nutrients(self):
         return results
 
     def ratings(self):
-        try:
-            ratings = self.schema.ratings()
-            if ratings > 0:
-                return ratings
-        except Exception:
-            pass
+        rating_elements = self.soup.find_all("p", {"class": "comment-rating"})
+        if not rating_elements:
+            return None
+
+        total_rating = 0
+        for element in rating_elements:
+            img_tag = element.find("img")
+            if img_tag and "src" in img_tag.attrs:
+                src = img_tag["src"]
+                match = re.search(r"(\d+)-star\.svg", src)
+                if match:
+                    total_rating += int(match.group(1))
```

Comment:
Small details again, but some additional indentation reduction opportunities here:

```suggestion
        total_rating = 0
        for element in rating_elements:
            img = element.find("img", {"src": True})
            if not img:
                continue
            match = re.search(r"(\d+)-star\.svg", img["src"])
            if match:
                total_rating += int(match.group(1))
```